keystone/files/mitaka/keystone.conf.Debian

```diff
@@ -951,7 +951,7 @@ domain_config_dir = /etc/keystone/domains
 {%- if server.get('backend', 'sql') == 'ldap' %}
 driver = ldap
 {%- else %}
-driver = sql
+driver = {{ server.get('identity', {}).get('backend', 'sql') }}
```

Comment:
Or only specify the options which are supported by the salt-formula.. (like an if/elseif/else structure there)